src/poliastro/plotting.py

```diff
@@ -399,6 +327,95 @@ def savefig(self, filename, **layout_kwargs):
         )
 
 
+class OrbitPlotter3D(_BaseOrbitPlotter):
+    """OrbitPlotter3D class.
+    """
+
+    def __init__(self):
+        super().__init__()
+        self._layout = Layout(
+            autosize=True,
+            scene=dict(
+                xaxis=dict(
+                    title="x (km)",
+                ),
+                yaxis=dict(
+                    title="y (km)",
+                ),
+                zaxis=dict(
+                    title="z (km)",
+                ),
+                aspectmode="data",  # Important!
+            ),
+        )
+
+    @u.quantity_input(elev=u.rad, azim=u.rad, distance=u.km)
+    def set_view(self, elev, azim, distance=5 * u.km):
+        x = distance * np.cos(elev) * np.cos(azim)
+        y = distance * np.cos(elev) * np.sin(azim)
+        z = distance * np.sin(elev)
+
+        self._layout.update({
+            "scene": {
+                "camera": {
+                    "eye": {
+                        "x": x.to(u.km).value, "y": y.to(u.km).value, "z": z.to(u.km).value
+                    }
+                }
+            }
+        })
+
+    def _redraw_attractor(self, min_radius=0 * u.km):
+        # Select a sensible value for the radius: realistic for low orbits,
+        # visible for high and very high orbits
+        radius = max(self._attractor.R.to(u.km), min_radius.to(u.km))
+        if radius < self._attractor_radius:
+            # If the resulting radius is smaller than the current one, redraw it
+            shape = _plot_sphere(radius, BODY_COLORS.get(self._attractor.name, "#999999"), self._attractor.name)
```

Comment:
This is almost exactly equal as `OrbitPlotter2D._redraw_attractor`, except for `_plot_sphere` vs `self._plot_circle`. Maybe we can have a `self._plot_sphere` in `OrbitPlotter3D`, rename `OrbitPlotter2D._plot_circle` to `OrbitPlotter2D._plot_sphere` (yes, it's a 2D sphere, right?) and we can move `_redraw_attractor` to the base class. What do you think?

Reply:
Yeah! True. It would be much better